Resources/Prototypes/_RMC14/Entities/Structures/Machines/Vending/Crew/requisitions.yml

```
@@ -39,8 +39,8 @@
       entries:
       - id: CMWeaponPistolMK80
         amount: 4
-      #- id: SU-6 Smart Pistol
-      #  amount: 3
+      - id: RMCWeaponPistolSU6
```

This is supposed to be three pistol cases that include the belt
```suggestion
      - id: RMCSmartPistolCase
```